src/style.css

```
@@ -1,3 +1,82 @@
+html {
+  font-family: "Roboto", sans-serif;
+  --color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  max-block-size: 1200px;
+  min-block-size: 1024px;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  padding-top: 15px;
+  display: flex;
+  height: 40px;
+  width: 40px;
+}
+
+.logo__image {
+  height: 100%;
+  width: 100%;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
```

```suggestion

.nav__item:not(:first-child) {
  margin-left: 20px;
}
```

FYI, we can simplify it